### ISSUE TITLE
Decide Upon Implementation of Adding Directory Levels instead of `Folder` Only

### ISSUE THREAD

**Issue by CodexLink** [CodexLink/chips.omp.json#8]
Refers to the following PR:
- https://github.com/JanDeDobbeleer/oh-my-posh/pull/3990.

I like the change, may _possibly_ add it from the upstream at the `oh-my-posh` repository. **But I need the following answer to this question that I did.**

### The Message
`@JanDeDobbeleer` may I ask if it would still be possible to create variants of the theme, I may plan on creating double-width (this was for compatibility issues with fonts like Iosevka Term wherein single space truncates the icons) and minimal (is for people who wanted lesser visual clutter, few icons, more strict context) variants just in case people have an issue with the current theme and prefer just having the other. I remembered that it is not allowed to post such themes because we have too many themes, but I'm not quite sure if variants are part of the restriction as they are also themes with few adjustments.

Let me know if there are concerns with the idea, as well as further suggestions and recommendations.

TL;DR: Basically if allowed to create another variant from the theme so that people have choices on such styling of the theme.

### Notes
- I don't have ETA, depending on the demand and response (_if seen_) I will do it as possible.
